docs-vim.js

```diff
@@ -8,6 +8,12 @@ vim = {
     }
 };
 
+var multiMaps = {
+    "b": [["ArrowLeft", true]],
+    "e": [["ArrowRight", true]],
+    "w": [["ArrowRight", true], ["ArrowRight", true], ["ArrowLeft", true]]
+};
+
```

Comment:
Can you add this as `vim.multiMaps`, e.g., on line 8? Just to keep all the configuration in one spot.

Reply:
Added in 85b4f01